src/index.css

```diff
@@ -23,6 +24,40 @@ img {
   max-width: 100%;
 }
 
+a {
+  color: #30323D;
+}
+
+a:hover{
+  color: #30323D;
+  text-decoration: none;
+}
+
+/* REGISTER */
+
+
+
+div.btn-control{
+  background: red;
+  display: flex;
+  justify-content: end;;
+}
+
+.form{
```

Comment:
Same here, if this is a FormWrap, make a styled component for it. Also make it explicit and clear why a form must be wrapped by this. Is it specific to one form or all forms?

Reply:
Are you going to remove the `.form` class from our index.css?